src/components/medialibrarycreator/medialibrarycreator.js

```diff
@@ -101,7 +125,10 @@ define(["loading", "dialogHelper", "dom", "jQuery", "components/libraryoptionsed
     }
 
     function onDialogClosed() {
-        loading.hide(), currentResolve(hasChanges)
+        // I can't see any corresponding call to loading.show,
+        // so I think this is not supposed to be here.
+        loading.hide();
```

Comment:
shouldn't you reset your `isCreating` or whatever flag is appropriate to `false` here?

Reply:
Okay, let's leave it be for now as-is.